spec/lib/whitehall_importer/integrity_checker/change_history_check_spec.rb

```diff
@@ -0,0 +1,99 @@
+RSpec.describe WhitehallImporter::IntegrityChecker::ChangeHistoryCheck do
+  describe "#match?" do
+    it "returns true if the proposed change history matches the Publishing API history for a non-draft edition" do
+      proposed_change_history = change_note("First published.", Date.yesterday.noon)
+      publishing_api_change_history = change_note("First published.", Date.yesterday.noon)
+
+
+      integrity_check = described_class.new(
+        true,
+        proposed_change_history,
+        publishing_api_change_history,
+      )
+      expect(integrity_check.match?).to be true
+    end
+
+    it "returns true if the proposed change history matches for a draft edition with mismatched time stamps for first item" do
+      proposed_change_history = [
+        {
+          "note" => "First published.",
+          "public_timestamp" => Date.yesterday.noon.rfc3339,
+        },
+        {
+          "note" => "Updated",
+          "public_timestamp" => Time.zone.now.rfc3339,
+        },
+      ]
```

Comment:
Could we use the `change_note` method to construct this?  I think you could call it with the required data twice and add them together.  Same would apply to the other instances of a change history with multiple change notes.